dockers/docker-router-advertiser/start.sh

```diff
@@ -1,11 +1,18 @@
 #!/usr/bin/env bash
 
-# Generate /etc/radvd.conf config file
-sonic-cfggen -d -t /usr/share/sonic/templates/radvd.conf.j2 > /etc/radvd.conf
-
 rm -f /var/run/rsyslogd.pid
 
 supervisorctl start rsyslogd
 
+# Router advertiser should only run on ToR (T0) devices
+DEVICE_ROLE=$(sonic-cfggen -d -v "DEVICE_METADATA.localhost.type")
+if [ "$DEVICE_ROLE" != "ToRRouter" ]; then
+    echo "Device role is not ToRRouter. Not starting router advertiser process."
+    exit 0
```

Comment:
Should this be before starting the rsyslogd?

Reply:
I want rsyslogd started so that supervisord can log the `echo` statement to the syslog. That way we have a record as to why radvd did not get started.